test/integration/targets/azure_rm_storageaccountkeys_facts/tasks/main.yml

```diff
@@ -0,0 +1,25 @@
+ - name: Create storage account name
+   set_fact:
+       storage_account: "{{ resource_group | hash('md5') | truncate(24, True, '') }}"
+
+ - name: Create new storage account
+   azure_rm_storageaccount:
+       resource_group: "{{ resource_group }}" 
+       name: "{{ storage_account }}"
+       account_type: Standard_LRS
+
+ - name: Get storage account keys
+   azure_rm_storageaccountkeys_facts:
+       resource_group: "{{ resource_group }}" 
+       name: "{{ storage_account }}"
+   register: output
+
+ - name: Assert that the result has Keys
+   assert:
+       that: output['keys'] | length > 0
```

Comment:
We should be testing more than this. Let's test the structure and that things like the name, resource_group, account_type and other known values are returned like we expect.